src/svg.panzoom.js

```diff
@@ -11,7 +11,7 @@ SVG.extend(SVG.Doc, SVG.Nested, {
   panZoom: function(options) {
     options = options || {}
     var zoomFactor = options.zoomFactor || 0.03
-    var zoomMin = options.zoomMin || 0
+    var zoomMin = options.zoomMin || Number.MIN_VALUE
     var zoomMax = options.zoomMax || Number.MAX_VALUE
```

Comment:
How about loosing the local variables and use `this` so that `panZoom` can be re-configured at any time?